services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -687,6 +756,76 @@ export const RateDetails = ({
                                                                 />
                                                             </FormGroup>
                                                         )}
+                                                        {showRatesAcrossSubs &&
+                                                            rateInfo.hasSharedRateCert && (
+                                                                <FormGroup
+                                                                    error={showFieldErrors(
+                                                                        rateErrorHandling(
+                                                                            errors
+                                                                                ?.rateInfos?.[
+                                                                                index
+                                                                            ]
+                                                                        )
+                                                                            ?.packagesWithSharedRateCerts
+                                                                    )}
+                                                                >
```

Comment:
Missing the label here: "Please select the submissions that also contain this rate certification." and View all submissions link